### ISSUE TITLE
Need More Documentation/Official announcements

### ISSUE THREAD

**Issue by jignesh248** [vergecurrency/VERGE#250]
Apologies for posting it here but there was no way to connect to DEVs. I totally understand devs time is precious but so is clarifying FUDs which if not done will deter verge from wide scale acceptance. I don't see a proper communication channel and thinks that if these questions and FUDs are well responded on official channel it would be good. 

A reddit post ( https://www.reddit.com/r/CryptoCurrency/comments/7ndzlk/verges_wraith_protocol_is_out_looked_at_the_code/ ) is widely being circulated and I have seen people putting verge on banlist in groups citing many reason one is that post and many more FUDs.

I would like to know, 
what does obfuscation of ledger mean?
if obfuscation of ledger is required or not?
If not why?
If yes is it not included in release?

Also if there are posts comparing verge to other privacy coins that would be good too. 

PS: Also a proper marketing channel to squash all FUDs would be best.
Please do not close this without a response, I am expecting a response and will circulate that around.

more FUDs : https://twitter.com/fluffypony/status/947878557702852608

Thanks

**Comment by besnikl** [comment]
Obfuscation is the practice of making something difficult to understand. Programming code is often obfuscated to protect intellectual property and prevent an attacker from reverse engineering a proprietary software program. 

In this case Obfuscation of an IP Address is to protect your privacy and not be able to track your location or whereabouts ... 

Cheers,
B.

**Comment by sasasavic82** [comment]
Seriously ... you should do your own research and make up your mind, not be a parrot reciting shit misinformed people say. Where were you for Christmas and NY?... cozy with your family ofcourse - while the Verge dev community punched away on producing a product for your lucrative benefit. I have a stake in this as much as the next person (and I bet, a lot more than you have), but you don't see me busting balls.

Either help, contribute, stop the FUD by informing yourself of what the product is about and what it's solving, or sell your holdings and move on.

The best "FUD squishing" is done by the supporters (in conjunction with the proper PR - which is being done by PROPER CHANNELS).

**Comment by bryanx86** [comment]
This coin has an amazing community. I love it!

**Comment by jignesh248** [comment]
@sasasavic82  This is what I am trying to do "Either help, contribute, stop the FUD by informing yourself "
There FUDs needs to be countered, I am trying to elicitate an official response. Not every other user is going to be a god level computer engineer, and clear the doubts by themselves, things sometimes needs to be communicated in layman language.

@besnikl Bro, I know obfuscation of IP addresses, I would like to understand more about the ledger part. What does obfuscation of ledger mean, I mean if wallet address will also be hidden/obfuscated or hidden then there wouldn't be a way to check if transaction is confirmed on blockchain. the author of the post doesn't make sense. Am I right or I am missing something?  Do any of the other coin provide obfuscation of ledger( in case if it something that I don't know ). I am a tech person, so in-depth tech explanation will work or some reading links.
